documentation-site/pages/blog/introducing-react-view/index.mdx

```diff
@@ -0,0 +1,272 @@
+import metadata from './metadata.json';
+import LiveEditor from './live-editor';
+import Layout from '../../../components/layout';
+import Yard from '../../../components/yard/index';
+import {BlogImage, Meta, Caption} from '../../../components/blog';
+
+import {View, PropTypes} from 'react-view';
+import {Button, SIZE} from 'baseui/button';
+
+export default Layout;
+
+<Meta data={metadata} />
+
+React View is an **interactive playground**, **documentation** and **code generator** for your components. You can use it to document your React component APIs, let users to live-edit the props and grab the generated ready-to-use source code.
+
+The example bellow demonstrates all basic features of React View:
+
+- A **live preview** of component. (`baseui/button` in this case)
+- Knobs representing `Button` props. They are **interactive**.
+- The **source code editor**. The source code is automatically generated based on the initial configuration and the state of playground. The source code is **also editable**.
+- Some additional actions. For example, you can **format the code with Prettier** or reset the playground state.
+
+<Yard
+  componentName="Button"
+  theme={[]}
+  props={{
+    children: {
+      value: 'Hello',
+      type: PropTypes.ReactNode,
+      description: 'Visible label.',
+    },
+    size: {
+      value: 'SIZE.default',
+      defaultValue: 'SIZE.default',
+      options: SIZE,
+      type: PropTypes.Enum,
+      description: 'Defines the size of the button.',
+      imports: {
+        'baseui/button': {
+          named: ['SIZE'],
+        },
+      },
+    },
+    onClick: {
+      value: '() => alert("click")',
+      type: PropTypes.Function,
+      description: 'Function called when button is clicked.',
+    },
+    disabled: {
+      value: false,
+      type: PropTypes.Boolean,
+      description: 'Indicates that the button is disabled',
+    },
+    overrides: {
+      value: undefined,
+      type: PropTypes.Custom,
+      description: 'Lets you customize all aspects of the component.',
+      custom: {
+        names: [],
+        sharedProps: {},
+      },
+    },
+  }}
+  scope={{
+    Button,
+    SIZE,
+  }}
+  imports={{
+    'baseui/button': {
+      named: ['Button'],
+    },
+  }}
+/>
+
+All these parts work in harmony. An update one of them synchronizes the state across all of them. So you can change the knob to update the source code or edit the source code to updat knobs.
+
+**React View is open sourced as [uber/react-view](https://github.com/uber/react-view)** and you can install with npm:
+
+```
+yarn add react-view
+```
+
+## Wait But Why?
+
+Our team is responsible for this website and the entire Base Web component library. Naturally, you might assume that most of our jobs consists of building React components. However, as the number of Base Web users grows, the reality is slightly different:
```

Comment:
Should we phrase this half-jokingly? I am a bit worried, that folks will take it too seriously

Reply:
Yea, this should be a joke. What would make it more obvious (without just completely giving it up by appending “j/k”)? :)